index.html

```diff
@@ -36,24 +36,236 @@ <h2>I'm a Software Developer</h2>
+              <h2 class="text">Multi-Post Stories Gain+Glory</h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="title">
+              <h2 class="text">Multi-Post Stories Gain+Glory</h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="title">
+              <h2 class="text">Multi-Post Stories Gain+Glory</h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="title">
+              <h2 class="text">Multi-Post Stories Gain+Glory</h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="title">
+              <h2 class="text">Multi-Post Stories Gain+Glory</h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="title">
+              <h2 class="text">Multi-Post Stories Gain+Glory</h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="about">
+        <h2>About me</h2>
+        <p>
+          I can help you build a product, feature or website. Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <button type="button" class="button">Get My Resume</button>
+
+        <div class="tools">
```

Comment:
Kindly update this element to use a `<ul>` tag, as it is one of the project requirements🙏.

<img width="686" alt="image" src="https://user-images.githubusercontent.com/37116322/219168802-9cd1f71f-48e1-45e6-85b9-38c7991e426d.png">

Also, ensure that each of the technology divs is also wrapped in `<li>` tags. You can do this by replacing each instance of `<div class="works">` with a `<li>` tag🫡